.vscode/extensions.json

```diff
@@ -3,7 +3,9 @@
     "charliermarsh.ruff",
     "ms-python.mypy-type-checker",
     "ms-python.python",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "redhat.vscode-yaml",
+    "fnando.linter"
```

Comment:
`trunk` also supports yamllint and seems to be a more robust extension. Reviews claim that linter can be quite slow.
https://marketplace.visualstudio.com/items?itemName=Trunk.io